src/fragments/lib/storage/flutter/remove.mdx

```diff
@@ -10,3 +14,29 @@ Future<void> deleteFile(String key) async {
   }
 }
 ```
+
+</Block>
+
+<Block name="Remove multiple objects (Developer Preview (Mobile, Web & Desktop))">
+
+Delete multiple objects uploaded to S3 by using `Amplify.Storage.removeMany` and specify the keys:
```

Comment:
I feel like we need to specify and add both the remove and removeMany APIs here. It was a bit confusing since I thought I can only use removeMany with the dev preview.

Reply:
Also can we change objects to files please?

Reply:
Updated the block switcher, and corrected the code example of `remove` usage in the developer preview.